libs/logic-apps-shared/src/utils/src/lib/models/uiInteractionData.ts

```diff
@@ -0,0 +1,10 @@
+import type { DiscoveryOpArray } from './operationDiscoveryResults';
+
+export interface UiInteractionData {
+  graphId?: string;
+  parentId?: string;
+  childId?: string;
+  nodeId?: string;
+  preloadedOperations?: DiscoveryOpArray;
+  dispatch?: any;
```

Comment:
What is the reason to pass the dispatcher here?

Reply:
Discussed offline. Removed the dispatcher here.